test/cppsuite/src/component/statistics/cache_limit.cpp

```diff
@@ -55,10 +55,11 @@ cache_limit::check(scoped_cursor &cursor)
         logger::log_msg(LOG_TRACE, name + " usage: " + std::to_string(use_percent));
 }
 
-std::string
-cache_limit::get_value_str(scoped_cursor &cursor)
+/* This loses precision but the metrics monitor requires an int64_t return type. */
+int64_t
+cache_limit::get_value(scoped_cursor &cursor)
 {
-    return std::to_string(get_cache_value(cursor));
+    return static_cast<int64_t>(get_cache_value(cursor));
```

Comment:
np:
Can we return int64_t from get_cache_value to avoid having a static_cast here?